scripts/src/main/resources/scripts/command/cobigen

```diff
@@ -10,10 +10,11 @@ then
   exit
 fi
 
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
-TOOL_VERSION_COMMAND="cg -v"
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
+TOOL_VERSION_COMMAND="cg -v"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
 
```

Comment:
After the installation the verification of the installation of cobigen is handled via `cobigen -v`, which gives 
`Unknown option: '-v'` as output. 
So I guess, as also suggested as possible solution by the devon ide, that the verification should be done by `cobigen --version` instead.

Reply:
I could not test this locally as I get this error in both cases:
https://github.com/devonfw/cobigen/issues/1591

However, I changed it as you suggested.